cloudformation.yml

```diff
@@ -12,90 +10,49 @@ Parameters:
       - staging
       - testing
 
-Mappings:
-
-  ChoiceTableCapacities:
-    development:
-      ReadMin: 1
-      WriteMin: 1
-    production:
-      ReadMin: 1
-      WriteMin: 1
-    staging:
-      ReadMin: 1
-      WriteMin: 1
-
-Globals:
-
-  Function:
-    Handler: index.handler
-    Runtime: python3.7
-    Tracing: Active
-    Timeout: 10
-    AutoPublishAlias: live
-    Environment:
-      Variables:
-        DYNAMODB_ENDPOINT: ''
-
+  EventName:
+    Type: String
+    Default: CONCERT_LAUSANNE_2019
 
 Resources:
 
-  DeadLetterQueue:
-    Type: AWS::SQS::Queue
+  UserPool:
+    Type: AWS::CloudFormation::Stack
     Properties:
-      QueueName: !Sub ${Environment}-dead-letter-queue
-      KmsMasterKeyId: alias/aws/sqs
+      TemplateURL: users/cloudformation.yml
+      Parameters:
+        Environment: !Ref Environment
+        EventName: !Ref EventName
 
-  ChoiceTable:
-    Type: AWS::DynamoDB::Table
+  Server:
+    Type: AWS::CloudFormation::Stack
     Properties:
-      TableName: !Sub ${Environment}-choice
-      AttributeDefinitions:
-        - AttributeName: songId
-          AttributeType: S
-        - AttributeName: listenId
-          AttributeType: S
-      KeySchema:
-        - AttributeName: songId
-          KeyType: HASH
-        - AttributeName: listenId
-          KeyType: RANGE
-      ProvisionedThroughput:
-        ReadCapacityUnits: !FindInMap [ChoiceTableCapacities, !Ref Environment, ReadMin]
-        WriteCapacityUnits: !FindInMap [ChoiceTableCapacities, !Ref Environment, WriteMin]
-      SSESpecification:
-        SSEEnabled: yes
-
-  ChoiceMadeSnsTopic:
-    Type: AWS::SNS::Topic
+      TemplateURL: lambdas/cloudformation.yml
+      Parameters:
+        AdminUserPoolGroupName: !GetAtt UserPool.Outputs.AdminUserPoolGroupName
+        CongitoUserPoolArn: !GetAtt UserPool.Outputs.CongitoUserPoolArn
```

Comment:
typo: congito (this is repeated in five other places further down)

Reply:
Good catch, thanks!